### DISCUSSION TITLE
Lesson 3: Fund Me

### DISCUSSION THREAD

**Discussion by ManiBharathiSS** [smartcontractkit/full-blockchain-solidity-course-py#1738] [Q&A] [answered]
while resetting the funders array why do we add ```(0)``` after ```new address[]``` 
```solidity
funders = new address[](0);
```
What does it mean?

**Comment by cromewar** [comment, marked as answer]
Hello @ManiBharathiSS 
This is because yo are reaching the pointer of the array, let's say the array have 5 elements, so you are accessing the first one with `(0)`.
I hope this info might help.

**Reply by ManiBharathiSS** [reply]
thanks @cromewar :)
